common/src/main/java/org/terracotta/angela/common/distribution/Distribution107Controller.java

```diff
@@ -196,17 +196,20 @@ public void stopTms(File installLocation, TerracottaManagementServerInstanceProc
             compile("^.*PID is (\\d+).*$"),
             mr -> javaPid.set(parseInt(mr.group(1))))
         .andTriggerOn(
-            compile("^.*\\QVote owner state: ACTIVE-COORDINATOR\\E.*$"),
```

Comment:
This was causing the voter process state from changing to STARTED.

IMO, all we need to check for is whether or not the voter was able to connect to the server in question. If the server doesn't accept the vote, the voter will never log "Vote owner state: ACTIVE-COORDINATOR"

Reply:
Maybe we should introduce a new state something like CONNECTED_TO_ACTIVE in addition to STARTED as tests might need to assert that state. 

Regarding test failure, voter should have connected to and logged `Vote owner state: ACTIVE-COORDINATOR`, we should investigate and fix it.

Reply:
Looks like the problem is with single node and voter configuration, not sure we need to fix it though. 

It would be better if we can make below changes to fix the test properly.

- [ ] Move the voter state to STARTED when the pid is found (like tsa server)
- [ ] Introduce new voter state (like CONNECTED_TO_ACTIVE) and move the state to it when `Vote owner state: ACTIVE-COORDINATOR` found
- [ ] Change the server count to 2 in the failing test

Reply:
Sure, will do.

Reply:
Pushed the change. Please see if the changes make sense.

Reply:
Not sure `Connected to` message signifies anything important here. Let's keep things simple for now, pid message moves to `STARTED` and active message moves to  `CONNECTED_TO_ACTIVE`. We can add more states later if needed.

Reply:
Done